### PR TITLE
988 no match status

### PR DESCRIPTION
**Pull request by Maxim-Lam**
### Description

List of proposed changes:

Added a LinkSummary component that displays a list of links. 

Display LinkSummary for no match results, showing three links provided in the user story.

### What to test for/How to test

1. enter false but valid data.
2. Test the links bring you to the appropriate places.

### Additional Notes

@s-laugh , @krischarbonneau  Please let me know if there is a specific way this should be styled and I will alter it. I left it pretty bare bones as to make it easy to change.

Also if you guys know of a better way to store the LinkSummaryItems outside of status.tsx, pls let me know.

### CODE REVIEW

File: pages/status.tsx [hunk 1]
```diff
@@ -26,6 +27,21 @@ const initialValues: CheckStatusRequestBody = {
...
+    },
+    {
+      href: 'https://www.canada.ca/en/immigration-refugees-citizenship/services/canadian-passports/contact-passport-program.html',
+      text: t('no-match-status-links.contact-program'),
+    },
+    {
+      href: 'https://eservices.canada.ca/en/reservation/',
+      text: t('no-match-status-links.appointment'),
+    },
+  ]
```
Comment (s-laugh) [inline pages/status.tsx:43]: I think you could have the whole object come from the translation. The links need to be translated too (french links for the french page)

File: pages/status.tsx [hunk 2]
```diff
@@ -118,14 +134,20 @@ const Status: FC = () => {
...
-          goBackStyle="primary"
-          checkAgainText={t('check-again')}
-        >
-          <p className=" mb-6 text-2xl">{t('unable-to-find-status')}</p>
-        </StatusInfo>
+        <>
+          <LinkSummary
+            id={'noMatchLinkSummary'}
+            links={noMatchHref}
+          ></LinkSummary>
```
Comment (s-laugh) [inline pages/status.tsx:141]: I think this should come after the status Info, as that would be the primary information the user is looking for.

File: components/LinkSummary.tsx [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+}
+
+export interface LinkSummaryProps {
+  id: string
+  links: LinkSummaryItem[]
+}
+
+const LinkSummary: FC<LinkSummaryProps> = ({ id, links }) => {
+  return (
+    <section id={id}>
```
Comment (s-laugh) [inline components/LinkSummary.tsx:16]: I'm not sure the id is adding any value to this component...

File: components/LinkSummary.tsx [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+      <ul className="list-none pb-6 ml-4">
```
Comment (s-laugh) [inline components/LinkSummary.tsx:17]: I think this should be styled like a normal list with bullets; and would be good to include a `p` tag with some text above, for now can give it the text "Other way to get your status:" (translated)

**Review by s-laugh — COMMENTED** [review]
This looks great! Answering your questions with a few suggestions.

File: components/LinkSummary.tsx (hunk 3 above)
Comment (Maxim-Lam) [reply components/LinkSummary.tsx:16]: My original thought process was maybe for some form of accessibility purpose or incase we needed to get the entire section by id for something but if it's not necessary I will remove it

File: pages/status.tsx [hunk 5]
```diff
@@ -26,6 +27,15 @@ const initialValues: CheckStatusRequestBody = {
...
   const { t } = useTranslation('status')
 
+  const getLinkSummaryItems: any = () => {
+    const LSI: any = t('no-match-status-links', { returnObjects: true })
+    const noMatchHref: LinkSummaryItem[] = []
+    for (const keys in LSI) {
+      noMatchHref.push(LSI[keys] as LinkSummaryItem)
+    }
+    return noMatchHref
+  }
```
Comment (s-laugh) [inline pages/status.tsx:37]: I don't think all this should be necessary, would just need to included `t('no-match-status-links', { returnObjects: true })` as the prop for the links...

**Review by s-laugh — APPROVED** [review]
Awesome!

**Pull request merged**